core/src/main/java/org/mule/runtime/core/internal/streaming/bytes/ManagedCursorStreamProvider.java

```diff
@@ -25,34 +26,36 @@
   /**
    * {@inheritDoc}
    */
-  public ManagedCursorStreamProvider(CursorContext cursorContext, CursorManager cursorManager) {
-    super(cursorContext, cursorManager);
+  public ManagedCursorStreamProvider(CursorContext cursorContext, CursorManager cursorManager,
+                                     MutableStreamingStatistics statistics) {
+    super(cursorContext, cursorManager, statistics);
   }
 
   /**
    * {@inheritDoc}
    */
   @Override
   protected CursorStream managedCursor(CursorStream cursor, CursorContext handle) {
-    return new ManagedCursorDecorator(cursor, handle);
+    return new ManagedCursorDecorator(cursor);
   }
 
   private class ManagedCursorDecorator extends CursorStream {
 
     private final CursorStream delegate;
-    private final CursorContext cursorContext;
+    //private final CursorContext cursorContext;
 
-    private ManagedCursorDecorator(CursorStream delegate, CursorContext cursorContext) {
+    private ManagedCursorDecorator(CursorStream delegate) {
       this.delegate = delegate;
-      this.cursorContext = cursorContext;
+      //this.cursorContext = cursorContext;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7643&issues=AWirZZpJotIXnmsUDp_y&open=AWirZZpJotIXnmsUDp_y)

[ISSUE_KEY]: AWirZZpJotIXnmsUDp_y